### ISSUE TITLE
Questions with --system-site-packages

### ISSUE THREAD

**Issue by opoplawski** [ionelmc/virtualenv#1]
Some comments/questions (please pardon my nievite)

Old virtualenv made a symbolic link for includes:
lrwxrwxrwx. 1 orion nwra 22 May  6 10:34 venv/include/python2.7 -> /usr/include/python2.7
This one has copies of the files, and a duplicate copy in venv/include/ and venv/local/include/

Old would make symbolic links to system .py files.  This one appears to copy in a large number of identical files, e.g.:

Files /scratch/orion/redhat/BUILDROOT/pyenv-ipython-3.1.0-1.fc22.x86_64/opt/pyenv/ipython-3.1.0/lib64/python2.7/zipfile.py and /usr/lib64/python2.7/zipfile.py are identical

**Comment by ionelmc** [comment]
Yes indeed, the _rewrite implementation_ errors on the side of copying too much stuff.

The plan is to add support for symlinks (for directories) and hardlinks (or symlinks where it doesn't break stuff) for files. It's just that the rewrite didn't got much traction so this feature is missing :disappointed: (you're the very first person to ask for this :grimacing:).

**Comment by techtonik** [comment]
Symlinks are evil. Not all filesystem support this (Vagrant shares no Windows). It took me a bit to remove them in Gratipay.
